docs/keycodes.md

```diff
@@ -165,6 +165,31 @@
 |`KC.RIGHT_ANGLE_BRACKET`|`KC.RABK`, `KC.GT` |`>`                |
 |`KC.QUESTION`           |`KC.QUES`          |`?`                |
 
+## [International Keys]
```

Comment:
Thanks for adding these back. I stripped them as they were separated out into a module, but I guess they never actually left due to another PR adding them back, but without the docs.